public/javascripts/conversation.js

```diff
@@ -318,4 +332,26 @@ $(function() {
         $('#conversation').collapse('show');
         event.preventDefault();
     });
+
+    $('#input-form').on('keydown', function(event) { // up button is pressed (For issue #172)
```

Comment:
No need to reference the issue in the comment, `git blame` will take care of that if we ever need it.